<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eyetracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_camera">
      <property name="title">
       <string>Camera</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_camera_hue">
         <property name="text">
          <string>Hue</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_camera_hue">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_camera_hue_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_camera_saturation">
         <property name="text">
          <string>Saturation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_camera_saturation">
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_camera_saturation_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_camera_brightness">
         <property name="text">
          <string>Brightness</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_camera_brightness">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_camera_brightness_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_camera_gain">
         <property name="text">
          <string>Gain</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="horizontalSlider_camera_gain">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_camera_gain_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_camera_contrast">
         <property name="text">
          <string>Contrast</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="horizontalSlider_camera_contrast">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_camera_contrast_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_camera_exposure">
         <property name="text">
          <string>Exposure</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="horizontalSlider_camera_exposure">
         <property name="minimum">
          <number>-20</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="label_camera_exposure_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox_general">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_general_blur">
         <property name="text">
          <string>Blur</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_general_blur">
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_general_blur_value">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_general_zoom">
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_general_zoom">
         <property name="maximum">
          <number>49</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_general_zoom_value">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_led">
      <property name="title">
       <string>LED</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_led_binary">
         <property name="text">
          <string>Binary Thresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_led_binary">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_led_binary_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_led_min">
         <property name="text">
          <string>Min Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_led_min">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_led_min_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_led_max">
         <property name="text">
          <string>Max Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_led_max">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_led_max_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox_pupil">
      <property name="title">
       <string>Pupil</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_pupil_binary">
         <property name="text">
          <string>Binary Thresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="horizontalSlider_pupil_binary">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_pupil_binary_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_pupil_min">
         <property name="text">
          <string>Min Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="horizontalSlider_pupil_min">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_pupil_min_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_pupil_max">
         <property name="text">
          <string>Max Size</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="horizontalSlider_pupil_max">
         <property name="minimum">
          <number>300</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_pupil_max_value">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_output">
      <property name="text">
       <string>Output to:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_output">
      <property name="text">
       <string>C:\eyetracking.dat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_output">
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_led_binary</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_led_binary_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_led_max</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_led_max_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_led_min</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_led_min_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_general_blur</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_general_blur_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_general_zoom</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_general_zoom_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pupil_binary</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_pupil_binary_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pupil_max</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_pupil_max_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_pupil_min</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_pupil_min_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_camera_hue</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_camera_hue_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_camera_saturation</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_camera_saturation_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_camera_brightness</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_camera_brightness_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_camera_gain</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_camera_gain_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_camera_contrast</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_camera_contrast_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_camera_exposure</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_camera_exposure_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
